components/Slider/Slider.js

```diff
@@ -42,6 +36,12 @@ export default class Slider {
           </div>
           : null
         }
+        <ReactSlider withBars
```

Comment:
There's [this one](https://github.com/seatgeek/react-slider) which has tests but is deprecated.
There's [this one](https://github.com/xeodou/react-range-slider) which has no tests (just like the one we're using).

Or there's [this one](https://github.com/whoisandie/react-rangeslider) which is tested and is 1.0 but doesn't seem to work at all on my iPhone.

Or we can make our own :)